docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,27 +1,195 @@
-= 5.1 Release Notes
-:description: In this release, we introduced pod scheduling support and added Phone Home service to collect usage data to shape our roadmap.
+= 5.2 Release Notes
 
-{description}
+In this release, we introduced Backup & Restore to/from PVC and hostPath
+features.
 
 == New features
 
-* Pod scheduling support for Hazelcast Custom Resource (Node Selector, Node Affinity, Pod Affinity&Anti-affinity, and Tolerations)
-```
+* Backup data into
+https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVC]
+(via
+https://docs.hazelcast.com/hazelcast/latest/storage/persistence[Hazelcast
```

Comment:
```suggestion
xref:5.1@hazelcast:storage:persistence.adoc[Hazelcast
```

We should use internal links for docs pages. Please apply this change to other docs links.